packages/main/src/themes-next/CustomListItem.css

```diff
@@ -15,14 +15,22 @@ ui5-li-custom .sap-phone .sapMLIB {
 }
 
 .sapMLIB.sapMCustomLI {
+	height: auto;
```

Comment:
I would go for height: 100%, because that way the bottom border will fall to the end of the list item

Reply:
I will revisit it once again, as Martin also suggested that providing min-height makes it not 100% custom (someone may want to set smaller height) and I will consider this comment as well

Reply:
I also thought about this case, but it becomes very difficult to handle, because the list can be SingleSelection or MultiSelection, which contains RadioButton or Checkbox, which are going to be cut.

Reply:
valid point

Reply:
> I would go for height: 100%, because that way the bottom border will fall to the end of the list item

Applied with the latest commit.